PurchasesCoreSwift/Support/ManageSubscriptionModalHelper.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ManageSubscriptionModalHelper.swift
+//
+//  Created by Andrés Boedo on 16/8/21.
+
+import StoreKit
+
+@available(iOS 9.0, *)
+@available(macOS 10.12, *)
+@available(watchOS, unavailable)
+@available(tvOS, unavailable)
+@objc public class ManageSubscriptionsModalHelper: NSObject {
```

Comment:
k, turns out this is really tricky to test. 
- we can't mock the UIApplication by doing inherit / replace, since creating an instance of UIApplication (or a subclass of it) crashes the test suite immediately with the message: "There can only be one UIApplication instance. (NSInternalInconsistencyException)". 
- replacing the `AppStore` type is really tricky because of its availability. 

While I don't love having something without automated tests, thoroughly testing this seems not worth the effort. I'll at least make sure that we call the right methods.

Reply:
gonna try this approach on Monday http://swiftyjimmy.com/mock-uiapplication-swift/, seems like it should help.